src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
Non breaking changes? That's not very open source of you :D

Remember the current is pre-alpha, so changes are to be expected

On Fri, Mar 13, 2015 at 9:41 AM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml
> <https://github.com/DashboardHub/PipelineDashboard/pull/75#discussion_r26373574>
> :
>
> > @@ -2,6 +2,10 @@ dashboardhub_app_homepage:
> >      path:     /
> >      defaults: { _controller: DashboardHubAppBundle:Default:index }
> >
> > +dashboardhub_app_dashboard:
> > +    path:     /d/{uid}
>
> Probably best, but also can't leave it too long as might confuse people or
> affect bookmarks - could keep old alive as well as new i guess & redirect -
> not ideal but an option
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/75/files#r26373574>
> .
>